packages/utilities/psammead-rich-text-transforms/src/candy-xml.js

```diff
@@ -6,7 +6,13 @@ const is = require('ramda/src/is');
 const { fragment, urlLink } = require('./models');
 
 const attributeTags = ['bold', 'italic'];
-const supportedXmlNodeNames = ['paragraph', 'link', 'url', ...attributeTags];
+const supportedXmlNodeNames = [
+  'paragraph',
+  'link',
+  'url',
+  'span',
```

Comment:
Was this the way that worked?  Or was it adding it to `attributeTags` that worked?  I'm worried that here we're creating a span block that Simorgh will ignore rather than pull the fragment out of... whereas in `attributeTags` we'll get a fragment with a span attribute, and then we'll just ignore the attribute?

Reply:
good spot. Just tested this and you are right. have updated and tested locally in simorgh with the generated output